persistence/src/test/scala/hmda/persistence/processing/HmdaFileParserSpec.scala

```diff
@@ -24,9 +24,9 @@ class HmdaFileParserSpec extends ActorSpec with BeforeAndAfterEach with HmdaFile
           | akka.loggers = ["akka.testkit.TestEventListener"]
           | akka.loglevel = DEBUG
           | akka.stdout-loglevel = "OFF"
-          | akka.persistence.journal.plugin = "akka.persistence.journal.inmem"
-          | akka.persistence.snapshot-store.plugin = "akka.persistence.snapshot-store.local"
-          | akka.persistence.snapshot-store.local.dir = "target/snapshots"
+          | akka.persistence.journal.plugin = "inmemory-journal"
```

Comment:
It looks like this block of config code is repeated in `HmdaFileValidatorSpec`. Could we extract the test config settings into a file?